client/my-sites/customer-home/main.jsx

```diff
@@ -197,6 +218,53 @@ const Home = ( {
 		return null;
 	};
 
+	const renderDnsSettingsDiagnosticNotice = () => {
+		if (
+			dismissedEmailDnsDiagnostics ||
+			isFetchingDomainDiagnostics ||
+			! emailDnsDiagnostics ||
+			emailDnsDiagnostics.code === 'domain_not_mapped_to_atomic_site' ||
+			emailDnsDiagnostics.all_essential_email_dns_records_are_correct
+		) {
+			return null;
+		}
+		const hasErrors = Object.values( emailDnsDiagnostics.records ).some(
+			( record ) => record.status === 'incorrect'
+		);
+
+		if ( hasErrors ) {
+			return (
+				<Notice
+					text={ translate(
+						// TODO: Add the corresponding support doc link when it's published
```

Comment:
We have the support doc now 🙂  https://wordpress.com/support/set-up-email-authentication-for-your-domain/